komodo-relational/src/main/java/org/komodo/relational/teiid/CachedTeiid.java

```diff
@@ -72,10 +89,10 @@
          */
         @Override
         public CachedTeiid create( final UnitOfWork transaction,
-                             final Repository repository,
-                             final KomodoObject parent,
-                             final String id,
-                             final RelationalProperties properties ) throws KException {
+                                   final Repository repository,
+                                   final KomodoObject parent,
+                                   final String id,
+                                   final RelationalProperties properties ) throws KException {
             return new CachedTeiidImpl(transaction, repository, id);
```

Comment:
* periodically reformat: certainly my preference, to remove silly little warnings and other noise. In our defence, the number of warnings though is miles lower than Designer ever was!
* Eclipse preferences are a hiding-to-no-where since they do cross devs preferred coding style, eg. using spaces around brackets, putting if/else single statements on same line. I think this is a no-win situation so suggest the following guidelines:
  * code that has been edited by commit can be reformatted to author's taste
  * code unaffected by commit should be left alone
  * turn off reformat-on-save

Following this will ensure that only modified code appears in commits and reduces the amount of unnecessary noise.